AFM: Docs for the April release (#29)
mulesoft/docs-functional-monitoring: modules/ROOT/nav.adoc

.xref:index.adoc[API Functional Monitoring]
* xref:index.adoc[About API Functional Monitoring]
* xref:afm-monitoring-public-apis.adoc[Monitoring the Endpoints of Public APIs]

> * xref:afm-monitoring-private-apis.adoc[Monitoring the Endpoints of Private APIs and Mixes of Both Public and Private APIs]

Reviewer: I'm sure that you tried, but, my, this is a long title!
Monitoring Private or Private Plus Public API Endpoints?

> * xref:afm-in-anypoint-platform.adoc[API Functional Monitoring in API Monitoring]

Reviewer: Won't just "API Functional Monitoring" do?

 ** xref:afm-create-monitor.adoc[Create a Monitor]
 ** xref:afm-ui-eval-create-monitor.adoc[Create a Monitor When Evaluating API Functional Monitoring]
 ** xref:afm-add-schedule.adoc[Add a Schedule]
 ** xref:afm-modify-schedule.adoc[Modify a Schedule]
 ** xref:afm-delete-schedule.adoc[Delete a Schedule]
 ** xref:afm-view-locations-and-schedules.adoc[View Your Locations and the Schedules That Use Them]
 ** xref:afm-run-test-now.adoc[Run a Test Immediately]

>  ** xref:afm-view-monitor-history.adoc[View the Details of Past Executions of a Test]

Reviewer: ```suggestion
 ** xref:afm-view-monitor-history.adoc[View the Details of Past Test Executions]
```

>  ** xref:afm-download-upload-monitor.adoc[Download a Copy of a Monitor, Upload a Test Suite as a Monitor, or Both]

Reviewer: We could shorten this title and then clarify in the text immediately following:
```suggestion
 ** xref:afm-download-upload-monitor.adoc[Download a Monitor Copy and Upload a Monitor Test Suite]
```
I understand that this could change your intended meaning but still want to suggest thinking about a workaround
